lake/ztests/manage-compact.yaml

```diff
@@ -0,0 +1,20 @@
+script: |
```

Comment:
I think this should live in cmd/zed/manage or cmd/zed/manage/compact until this is more mature.

Reply:
ok